ui/components/multichain/global-menu/global-menu.js

```diff
@@ -101,7 +102,7 @@ export const GlobalMenu = ({ closeMenu, anchorElement }) => {
   ///: END:ONLY_INCLUDE_IN
 
   return (
-    <Menu anchorElement={anchorElement} onHide={closeMenu}>
+    <Popover referenceElement={anchorElement} isOpen={closeMenu}>
```

Comment:
`isOpen` is a boolean type while `closeMenu` is a function type I think the equivalent prop is `onBlur` for the Popover component

```suggestion
    <Popover referenceElement={anchorElement} onBlur={closeMenu}>
```